tests/hscRepository.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+#
+# Copyright 2008-2015  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import os
+import unittest
+from glob import glob
+from shutil import rmtree
+from subprocess import check_call
+from tempfile import mkdtemp
+
+import lsst.afw.geom as afwGeom
+import lsst.daf.persistence as dafPersist
+import lsst.pex.exceptions as pexExcept
+import lsst.utils.tests as utilsTests
+from lsst.utils import getPackageDir
+
+TEST_DATA_PACKAGE = "testdata_subaru"
+
+def requirePackage(packageName):
+    """
+    Decorator to skip unit tests unless the package ``packageName`` is set up.
+    """
+    try:
```

Comment:
Necessary? No, but I think it's more readable in the code, and it shouldn't make any difference to the processed documentation (per PEP 0257).